rclpy/threading/package.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>examples_rclpy_threading</name>
+  <version>0.0.0</version>
+  <description>Package with examples of using threading with rclpy.</description>
+  <maintainer email="dhood@osrfoundation.org">D. Hood</maintainer>
+  <license>Apache License 2.0</license>
+
+  <exec_depend>python3-matplotlib</exec_depend>
```

Comment:
I have made a note so that this is clear to others looking at the code in https://github.com/ros2/examples/pull/144/commits/553ca84bbed9e1fea36cb5ccdf734af226d3f111